packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -61,12 +61,15 @@ export default function SidebarNavigationScreenNavigationMenu() {
 	const _handleSave = ( edits ) => handleSave( navigationMenu, edits );
 	const _handleDuplicate = () => handleDuplicate( navigationMenu );
 
+	const backPath = { path: '/navigation', postId };
```

Comment:
I think this is an error (although it may already exist on `trunk`). 

When going back from a single Navigation Menu you should return to the listing route which is `/navigation`. The postId should not be required for this.

This will alter slightly once https://github.com/WordPress/gutenberg/pull/61275 is merged however...

Reply:
This is correct and intended. This `backPath` does return you to to listing route which is `/navigation`. The `/navigation` route is in the `path` field. The additional `postId` specifies that we are returning from the editor for `postId` and that `postId` should be selected in the listing view.

In the navigations listing this is unused, but look how a DataView-enabled `/page` listing looks like with the `postId` parameter:

<img width="1381" alt="Screenshot 2024-05-01 at 16 42 54" src="https://github.com/WordPress/gutenberg/assets/664258/46871583-2014-4510-b339-2732dceac982">

Here the `postId=58` page named "Shop" is selected, and we are on a listing page, not on an editor page for "Shop".

For consistency, the `postId` is included in all "listing route" backpaths no matter whether they are actually used.

Reply:
Yes but that assumes that Navigation uses dataviews which as yet they don't because we have no good UX for that.

Feel free to merge this "as is". Someone will just end up resolving this in the PR I referenced.

Reply:
What's the flow for me to load the screen where this sidebar loads?

I also don't think we should have in the URL parameters we don't need, but I want to double check I'm not missing something.

Reply:
OK, I removed the `postId` parameter from the `/navigation` link, and also from the `/wp_template` link. They were harmless, but still redundant. We can add them back at the moment when the listing UI actually has use for them.